mteb/tasks/STS/por/SickBrSTS.py

```diff
@@ -25,7 +25,7 @@ class SickBrSTS(AbsTaskSTS):
         date=("2018-01-01", "2018-09-01"),  # rough estimate
         domains=["Web", "Written"],
         task_subtypes=["Textual Entailment"],
-        license="unknown",
+        license="Unknown",
```

Comment:
What is the difference between "Unknown" and " Not Specified"? Is there a meaningful distinction here?

Reply:
No we should standardize them to "Not specified"